CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 # Upcoming
 
 ### 🔄 Changed
+- Improve performance of `ChatChannel` database model conversions more than 4 times [#3322](https://github.com/GetStream/stream-chat-swift/pull/3322)
```

Comment:
You forgot to put the StreamChat header and also we can use the "Performance" subheader for performance improvements. We have done this in the past as well 👍 
```suggestion
## StreamChat
### ⚡ Performance
- Improve performance of `ChatChannel` database model conversions more than 4 times [#3322](https://github.com/GetStream/stream-chat-swift/pull/3322)
```